scripts/setup/apt-repos/zulip/custom.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
I think a block comment here explaining what this script does would be worthwhile, since it's a little dense.

I believe it to be the script for adding custom apt repositories (Currently, just the PGroonga apt repositories), including gpg keys.